core/src/main/java/org/ocpsoft/prettytime/units/Minute.java

```diff
@@ -17,6 +17,10 @@
 
 import org.ocpsoft.prettytime.impl.ResourcesTimeUnit;
 
+/**
+ * @deprecated Use {@link org.ocpsoft.prettytime.ResourcesChronoUnit#MINUTES} instead.
+ */
+@Deprecated
```

Comment:
Please remove these deprecations. ChronoUnit should simply be an alternate method of configuration.